charts/backstage/values.yaml

```diff
@@ -111,7 +114,7 @@ upstream:
 
     args:
       # This additional `app-config`` file is generated by the initContainer below, and contains the merged configuration of installed dynamic plugins.
-      - '--config'
+      - "--config"
```

Comment:
These should stay single quotes: https://support.asg.com/mob/mvw/10_0/mv_ag/using_quotes_with_yaml_special_characters.htm#:~:text=Single%20quotes%20let%20you%20put,returned%20as%20the%20string%20%5Cn.&text=Double%20quotes%20parse%20escape%20codes,as%20a%20line%20feed%20character.